common/address_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build TESTNET
```

Comment:
temporarily added this build flag to prevent wrong build tags in unit tests. 

Will adjust here once the consolidation of build flags happen.